previous-versions.md

```diff
@@ -0,0 +1,203 @@
+---
+title: PyTorch | Previous Versions
+id: previous-versions
+permalink: /previous-versions/
+layout: about
+---
+
+# Installing previous versions of PyTorch
+
+We'd prefer you install the [latest version](http://pytorch.org/),
+but old binaries and installation instructions are provided below for
+your convenience.
+
+### Via conda
+
+To install a previous version of PyTorch via Anaconda or Miniconda,
+replace "0.1.12" in the following commands with the desired version
+(i.e., "0.2.0").
+
+Installing with CUDA 8
+
+`conda install pytorch=0.1.12 cuda80 -c soumith`
+
+Installing with CUDA 7.5
+
+`conda install pytorch=0.1.12 cuda75 -c soumith`
+
+Installing without CUDA
+
+`conda install pytorch=0.1.12 -c soumith`
+
+### From source
+
+It is possible to checkout an older version of [PyTorch](https://github.com/pytorch/pytorch)
+and build it.
+You can list tags in PyTorch git repository with `git tag` and checkout a
+particular one (replace '0.1.9' with the desired version) with
+
+`git checkout v0.1.9`
+
+Follow the install from source instructions in the README.md of the PyTorch
+checkout.
+
+### Via pip
+
+Download the `whl` file with the desired version from the list below, and run
+
+`pip install /path/to/whl/file`
+
+You can also directly pass a URL to pip:
+
+`pip install http://download.pytorch.org/whl/cu80/torch-0.2.0.post3-cp36-cp36m-manylinux1_x86_64.whl`
+
+CUDA support is optional; for example, you can install the
+cu80/torch-0.2.0.post3-cp36-cp36m-manylinux1_x86_64.whl package on a system
+that does not have CUDA.
```

Comment:
I'm not sure if this is true: how does one install pytorch on a linux system that doesn't have CUDA?

The selection tool on the website just says to install the whl under cu75 -- does it work for the cu80 whl too?

Reply:
yes it works with cuda80 wheel too.
Basically, we ship with CUDA bits by default. The CUDA bits dont try to initialize GPU context right away. For that we have some guards to see if a NVIDIA driver exists, and then only activate CUDA bits.
Here's relevant code: https://github.com/pytorch/pytorch/blob/master/torch/cuda/__init__.py#L27-L70